application/src/screens/Cooking.tsx

```diff
@@ -298,69 +236,80 @@ export function Cooking({ navigation, route }: Props) {
           />
         );
         break;
-      case "extendOrFinish":
-        taskConfirmButtons = (
-          <TaskConfirm
-            confirmType={taskConfirmType}
-            onFinishPress={() => {
-              if (okToPress) {
-                let t = activeTask;
-                if (t) {
-                  scheduler.finishPassiveTask(t);
-                }
-                setOkToPress(false);
-                setTimeout(() => setOkToPress(true), OK_TIME_BETWEEN_CLICK);
-              }
-            }}
-            onExtendPress={() => {
-              if (okToPress) {
-                let t = activeTask;
-                if (t) {
-                  scheduler.extendPassive(t);
-                }
-                setVisiblePassiveTasks((prevVisablePassiveTasks) =>
-                  prevVisablePassiveTasks.filter((task) => task !== t)
-                );
-                setOkToPress(false);
-                setTimeout(() => setOkToPress(true), OK_TIME_BETWEEN_CLICK);
-              }
-            }}
-          />
-        );
+      case "unavailable":
+        taskConfirmButtons = <TaskConfirm confirmType={taskConfirmType} />;
         break;
-      case "interruptOrContinue":
-        taskConfirmButtons = (
-          <TaskConfirm
-            confirmType={taskConfirmType}
-            //klickat på interrupt/stäng av i förväg
-            onFinishPress={() => {
-              if (okToPress) {
-                let t = activeTask;
-                if (t) {
-                  scheduler.finishPassiveTask(t);
-                }
-                setOkToPress(false);
-                setTimeout(() => setOkToPress(true), OK_TIME_BETWEEN_CLICK);
-              }
+    }
+
+    let timersComponent: JSX.Element;
+    if (earliestInstr) {
+      timersComponent = (
+        <View
+          style={{
+            justifyContent: "center",
+            marginLeft: 10,
+            maxWidth: "70%",
+          }}
+        >
+          <CookingTimer
+            onPress={() => setTimerModalVisible(true)}
+            finish={earliestTimer}
+            displayRemainingTime="shown"
+            size="small"
+          />
```

Comment:
Det ser lite konstigt ut på min mobil.

![old](https://user-images.githubusercontent.com/16006944/116548451-87471000-a8f4-11eb-807c-526f15d6ad2d.png)

Den verkar se det som att texten "tar upp mer plats än den faktiskt tar upp".
Att lägga till negativa "margin" är ett hack som fixar det, iallafall på min mobil.

![new](https://user-images.githubusercontent.com/16006944/116548630-cf663280-a8f4-11eb-92e8-510631974072.png)


```suggestion
          <View style={{ marginBottom: -8, marginTop: -8, }}>
            <CookingTimer
              onPress={() => setTimerModalVisible(true)}
              finish={earliestTimer}
              displayRemainingTime="shown"
              size="small"
            />
          </View>
```